CHANGELOG.md

```diff
@@ -3,7 +3,29 @@
 ## Unreleased
 
 ### Breaking changes
-You must make the following changes when you migrate to this release, or your service may break.
+Where applicable, you must make the following changes when you migrate to this release, or your service may break.
+
+#### Check your accordions still display as expected
+
+[See the new design and revised guidance for accordions in the Design System](https://design-system.service.gov.uk/components/accordion/). In particular, review the guidance on when and when not to use accordions.
+
+You should check that your accordions still display as expected if you:
+
+- have applied any custom styles to your accordions
+- use accordions in non-standard ways
+- have section headings that contain unusual formatting (for example, wrappers)
+
+You do not need to make changes to the content that accordions hide or show - this release's changes do not affect it.
+
+These changes were introduced in [pull request #2257: Update accordion design](https://github.com/alphagov/govuk-frontend/pull/2257).
+
+#### Only use the accordion’s summary line for ‘phrasing content’
```

Comment:
On the one hand, it makes sense to talk about this as it introduces an actual technical constraint.

However I'm a bit worried that:
- this highlights an edge case that will impact only a very small number of users or even none at all (we haven't seen any evidence that teams put non-phrasing content inside the summary line)
- the more common scenario for needing to limit the length of the summary line might be get overlooked

I've left a comment [here](https://github.com/alphagov/govuk-design-system/pull/1961/files#r747756914) to discuss with @calvin-lau-sig7 whether these changes (calls to action) users might need to make should be here or in the guidance.

Reply:
@hannalaakso
Re edge case
The first sentence (line 6 above) in the 'Breaking changes' section is:

> Where applicable, you must make the following changes when you migrate to this release, or your service may break.

Does that "Where applicable" get across that users only need to do something if the change affects their service?

Re length of summary line, I've added content below (line 28 onwards).

Reply:
@EoinShaughnessy Re: 'edge case', we probably don't need to use those words specifically.

However, could we swap these two sections around: 

'## Only use the accordion’s summary line for ‘phrasing content’'
and 
'#### Keep summary line content brief'

This way the more common use case will come first.

Reply:
Thanks @hannalaakso! That makes sense. Have swapped around the order.